test.py

```
@@ -1,5 +1,41 @@
+import os
+import sys
 
-print c
\ No newline at end of file
+def process(x):
+    return x * 2
+
+def do_something():
+    a = 10
+    b = 20
+    c = 30
```

Ruff (0.12.2)

Local variable `c` is assigned to but never used

Remove assignment to unused variable `c`

(F841)